docs/tutorials/quickstart-app-auth.md

```diff
@@ -3,20 +3,17 @@ id: quickstart-app-auth
 title: Monorepo App Setup With Authentication
 ---
 
-###### September 15th 2020 - @backstage/create-app - v0.1.1-alpha.21
+###### January 8th 2021 - @backstage/create-app - v0.4.5
 
 <br />
 
 > This document takes you through setting up a Backstage app that runs in your
 > own environment. It starts with a skeleton install and verifying of the
-> monorepo's functionality. Next, GitHub authentication is added and tested.
+> monorepo's functionality. Next, authentication is added and tested.
 >
 > This document assumes you have Node.js 12 active along with Yarn and Python.
```

Comment:
The recommended version is Node 14 now. I wouldn't expect you to retest the whole thing but it might be nice to say "Node.js 12 or 14 active".

Reply:
:+1: Good point. I'll add that in.